woocommerce/google-listings-and-ads
# Add Connection test for Api Pull Notification request

### Changes proposed in this Pull Request:

This PR adds a section in the connection test page for Notifying an item with a topic.


### Screenshots:

<img width="811" alt="Screenshot 2024-01-26 at 09 47 26" src="https://github.com/woocommerce/google-listings-and-ads/assets/5908855/70260991-6443-40d3-add5-86ed0b2edefb">



### Detailed test instructions:


1. Go to Connection Test page
2. See the new section Partner API Pull Integration
3. Send a request with a number and a topic


### Additional details:

- Validation of item (product/copuon) will be done in future PRs


### Changelog entry
